test/suite/integration/probs.config.js

```diff
@@ -26,7 +27,8 @@ const prepDevice = async (device) => {
 };
 
 module.exports = {
-  async setupFile() {
+  async setupFile(file) {
+    console.log("[PREP]", file);
```

Comment:
make sure you remove the logs

Reply:
Those are strictly for testing. They show when a new file is processed. :)